pandas/tests/io/json/test_pandas.py

```diff
@@ -1242,6 +1243,25 @@ def test_read_jsonl_unicode_chars(self):
         expected = DataFrame([["foo\u201d", "bar"], ["foo", "bar"]], columns=["a", "b"])
         tm.assert_frame_equal(result, expected)
 
+    def test_to_json_large_numbers(self):
+        # GH34473
+        bigNum = sys.maxsize + 1
+
+        originalSeries = Series(bigNum, dtype=object, index=["articleId"])
+        originalSeries.to_json()
+        # json = StringIO(json)
+        # newSeries = read_json(json, typ="series")
+        # tm.assert_series_equal(originalSeries, newSeries)
+
+        originalDataFrame = DataFrame(
+            bigNum, dtype=object, index=["articleId"], columns=[0]
+        )
+        originalDataFrame.to_json()
```

Comment:
It's a little tedious now since we can't roundtrip these but can you manually construct the expected JSON string and compare the result to that?

Reply:
For sure!